docs/home/contribute/localization.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Localizing Kubernetes Documentation
+---
+
+We're happy to add localizations of Kubernetes documentation to the website!
+
+Localizations must meet the following requirements for _workflow_ (how to localize) and _output_ (what to localize).
+
+## Workflow  
+
+- All localization work must happen within a Kubernetes organization repository specifically dedicated to localization.
+
+    For example: https://github.com/kubernetes/kubernetes-docs-cn
+
+    To open a localization repository, [contact the SIG docs lead](https://kubernetes.slack.com/messages/C1J0BPD2M) on Slack for assistance.
+
+- Teams must track their overall progress with a GitHub project.
+
+    Projects must include columns for pending work, work in progress, and completed work.
+
+- L10n teams must provide their own repository maintainers.
+
+- All localization work must be self-sustaining with the team's own resources.
+
+- Upstream contributions are limited to a maximum of one PR with squashed commits per week.
```

Comment:
L10n teams must ensure that every localized page is QAed by a separate documentation person from the one who did the translation. There is a strong preference that the translator and QA person be from different companies to ensure an independent and unbiased review.